tests/topotests/zebra_startup_speeds/test_zebra_startup_speeds.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+
+"""
+test_startup_of_speeds.py: Test router startup with multiple interfaces
+"""
+
+import os
+import sys
+import json
+import pytest
+from functools import partial
+
+CWD = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(CWD, "../"))
+
+# pylint: disable=C0413
+from lib import topotest
+from lib.topogen import Topogen, TopoRouter, get_topogen
+from lib.topolog import logger
+from lib.common_config import step
+from time import sleep
+
+
```

Comment:
should this have the zebra "pytestmark" - or are we not maintaining those anymore?

Reply:
there is no testing without zebra.  As such we never created a pytestmark for zebra since you need to run all the tests when you change zebra in any event.